Python.xs

```diff
@@ -22,22 +22,29 @@
 
 #ifdef CREATE_PYTHON
 void do_pyinit() {
-#ifdef EXPOSE_PERL
-    PyObject *main_dict;
-    PyObject *perl_obj;
```

Comment:
Moving these declarations raises the requirement for the compiler to C99. While that sounds like stone-age, it's actually been only a few years since Microsoft finally added support for C99 to their compiler. Even on compilers that do support C99, it may require a command line argument like `-std=C99`. Granted, I don't actually know whether the rest of the code is "clean" or if it's already too late (I personally tend towards sloppiness in that regard), but since this is a purely incidental change, unrelated to the actual problem this commit solves, I'd rather take the safe path.

Reply:
I have moved all declaration to to top.